dockers/nvidia/Dockerfile

```
@@ -12,8 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+ARG PYTORCH_VERSION=21.07
```

There isn't a 21.07 PyTorch version, the Nvidia docs say "PyTorch Release" 
```suggestion
ARG PYTORCH_RELEASE=21.07
```

---

yep, good point :]